hhs_oauth_server/views/openapi.py

```diff
@@ -0,0 +1,16 @@
+import yaml
```

Comment:
I need to move where this lives, since it's not a case of an implementation specific to bluebutton it should go in an `app`

Reply:
:thumbsup: